21:10:46.813 INFO Test execution started: project/add_page
21:10:46.813 INFO Browser: chrome
21:10:46.813 INFO Environment: test
21:10:46.813 INFO Using data: 
    url: http://localhost:8000

21:10:48.933 INFO Navigate to: 'http://localhost:8000'
21:10:49.038 INFO Write 'admin' in element username
21:10:49.121 INFO Write 'admin' in element password
21:10:49.160 INFO Click login_button
21:10:49.614 INFO Random value generated: wqfcl
21:10:49.614 INFO Store value wqfcl in key page_name
21:10:49.864 INFO Write 'wqfcl' in element None
21:10:49.864 INFO Press key: ENTER
21:10:49.918 INFO Assert that 0 equals 1
21:10:49.919 INFO Take screenshot error
21:10:50.459 ERROR An error ocurred:
Traceback (most recent call last):
  File "/Users/lucianofeo/Desktop/golem/src/golem/test_runner/test_runner.py", line 115, in run_test
    test_module.test(execution.data)
  File "/Users/lucianofeo/Desktop/golem/example/projects/golem/tests/project/add_page.py", line 16, in test
    project.verify_page_exists(data.page_name)
  File "/Users/lucianofeo/Desktop/golem/example/projects/golem/pages/project.py", line 88, in verify_page_exists
    actions.assert_equals(len(tree_elements), 1)
  File "/Users/lucianofeo/Desktop/golem/src/golem/actions.py", line 56, in assert_equals
    raise Exception('Expected {} to equal {}'.format(actual_value, expected_value))
Exception: Expected 0 to equal 1
21:10:50.460 INFO Close driver
